WalletWasabi/Tor/TorProcessManager.cs

```diff
@@ -60,82 +62,102 @@ public TorProcessManager(TorSettings settings, EndPoint torSocks5EndPoint)
 
 		private CancellationTokenSource Stop { get; set; }
 
-		public void Start(bool ensureRunning)
+		/// <summary>
+		/// Installs Tor if it is not installed, then it starts Tor.
+		/// </summary>
+		/// <param name="ensureRunning">
+		/// If <c>false</c>, Tor is started but no attempt to verify that it actually runs is made.
+		/// <para>If <c>true</c>, we start Tor and attempt to connect to it to verify it is running (at most 10 attempts).</para>
```

Comment:
Should this say `(at most 25 attempts)`?

Reply:
Yes